src/app/app-common/PageSource/include/OpenKneeboard/PlainTextPageSource.h

```diff
@@ -66,18 +69,21 @@ class PlainTextPageSource final : public IPageSource {
   std::vector<std::vector<winrt::hstring>> mCompletePages;
   std::vector<winrt::hstring> mCurrentPageLines;
   std::vector<std::string> mMessages;
```

Comment:
Now we have `mAllMessages`, `mMessages` should be renamed for clarity - perhaps `mMessagesToLayout` ?

If you're using vscode, you can *probably* do this just by moving the cursor over here then hitting F2

Reply:
also, see previous note - perhaps `std::string_view` is better here